jerry-core/CMakeLists.txt

```diff
@@ -25,220 +52,174 @@ project (JerryCore C ASM)
                   WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
                   OUTPUT_VARIABLE JERRY_GIT_COMMIT
                   OUTPUT_STRIP_TRAILING_WHITESPACE)
- # Get build date
-  execute_process(COMMAND date +%d/%m/%Y
-                  OUTPUT_VARIABLE JERRY_BUILD_DATE
-                  OUTPUT_STRIP_TRAILING_WHITESPACE)
-
+else()
+  set(JERRY_GIT_BRANCH "UNDEFINED")
+  set(JERRY_GIT_COMMIT "UNDEFINED")
+endif()
+
+# Get build date
+execute_process(COMMAND date +%d/%m/%Y
+                OUTPUT_VARIABLE JERRY_BUILD_DATE
+                OUTPUT_STRIP_TRAILING_WHITESPACE)
+
+set(DEFINES_JERRY
+    ${DEFINES_JERRY}
+    JERRY_ENABLE_SNAPSHOT_SAVE
+    JERRY_ENABLE_SNAPSHOT_EXEC
```

Comment:
Yes, it does. This behaviour is similiar to the current one.
But this is unnecessary, so we made this defines optionally.
If both of them are disabled, then the size of Thumb-2 binary on RPi2 is reduced by ~3.5Kbyte.